hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/ContainerDBServiceProviderImpl.java

```diff
@@ -185,7 +185,7 @@ public long getKeyCountForContainer(Long containerID) throws IOException {
    */
   @Override
   public boolean doesContainerExists(Long containerID) throws IOException {
-    return containerKeyCountTable.get(containerID) != null;
+    return containerKeyCountTable.isExist(containerID);
```

Comment:
Can we move this change to a new Jira, as this does not belong to this Jira issue?